core/cautils/getter/getpolicies.go

```diff
@@ -38,3 +39,7 @@ type IBackend interface {
 type IControlsInputsGetter interface {
 	GetControlsInputs(clusterName string) (map[string][]string, error)
 }
+
+type IAttackTracksGetter interface {
```

Comment:
Do we still need a separated interface?

Reply:
Because it behaves differently at the moment... if we choose to extend the IPolicyGetter we need to make some additional changes